components/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/APIConsumer.java

```diff
@@ -38,6 +38,15 @@
      */
     public BusinessOwner getBusinessOwner(int businessOwnerId) throws AppManagementException;
 
+    /**
```

Comment:
Formatting and Java doc comments issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files